lib/matplotlib/animation.py

```diff
@@ -349,6 +350,7 @@ def _run(self):
             output = sys.stdout
         else:
             output = subprocess.PIPE
+        command = [fspath_no_except(cmd) for cmd in command]
```

Comment:
or even better to matplotlib.compat.{Popen,check_output}:
```
subprocess.Popen([Path("ls"), Path("-a"), Path(".")])
```
works (even though this is arguably silly...)